pyEpiabm/pyEpiabm/routine/file_population_config.py

```diff
@@ -93,8 +96,14 @@ def make_pop(input_file: str, random_seed: int = None):
             for column in input.columns.values:
                 if hasattr(InfectionStatus, column):
                     value = getattr(InfectionStatus, column)
-                    new_microcell.add_people(int(line[column]),
-                                             InfectionStatus(value))
+                    for i in range(int(line[column])):
+                        person = Person(new_microcell)
+                        person.infection_status = InfectionStatus(value)
+                        new_microcell.add_person(person)
+                        host_sweep.update_next_infection_status(person)
+                        host_sweep.update_time_status_change(person, 0)
+                        if str(person.infection_status).startswith('Infect'):
+                            HostProgressionSweep.set_infectiousness(person, 0)
```

Comment:
I am a bit confused: If the person's infectious status is a type of infectious, why would you set the infectiousness to 0 in the host progression sweep?

Reply:
This had me confused for ages as well! Turns out that zero isn't the infectiousness, its the time you are setting the value at - the actual value is sampled from infectiousness. I've tidied it up to make that more obvious though